### DISCUSSION TITLE
Help: how to customize device?

### DISCUSSION THREAD

**Discussion by holefrog** [sle118/squeezelite-esp32#393] [General]
Hello,

I am interested in adding sound effects such as power on and off indicators to my player, to help identify its status while using an IR remote.

I am aware of the option to customize devices through LMS--Advanced--Customize Device. However, after adding my sound effects there, I am unable to locate the menu in the player to configure it.

I followed the instructions: "These optional sounds will be available from the Settings->Advanced->Sound Effects menu on your controller." While I did find an option labeled "Sounds & Effects" in the player settings, it does not seem to be the correct one. I cannot locate the "Advanced" option under Settings.

Any assistance would be greatly appreciated.

**Comment by sle118** [comment]
I guess this is probably more a LMS question that would be better asked on the Logitech forums, but maybe someone here had an answer?
